internal/errors/migrated_errors.go

```diff
@@ -13,16 +13,19 @@ import (
 
 // Standard error variables for various DiceDB-related error conditions.
 var (
```

Comment:
Hi @vishnuchandrashekar any idea what has changed in this file. Need not to change unless adding any new error type.

Reply:
Hi, I've added the following error types: ErrInvalidBitfieldType, ErrBitfieldOffset, ErrOverflowType. Should I remove them and use `diceerrors.ErrGeneral`?

Reply:
Ohh got it. Yes you can use General Errors since bit errors and even Overflow for a bit will not get used at anywhere else.